hack/mkdocs-generate-conformance.py

```diff
@@ -50,28 +50,49 @@ def on_pre_build(config, **kwargs):
                                                 'HTTPRoutePathRedirect', 'HTTPRouteHostRewrite', 'HTTPRoutePathRewrite', 'HTTPRouteRequestMirror', 'HTTPRouteRequestMultipleMirrors', 'HTTPRouteRequestTimeout', 'HTTPRouteBackendTimeout', 'HTTPRouteParentRefPort']
 
 
+# NOTE add past versions of implementations to here when new release is cut
+past_versions = """
+??? info
+
+
+    The latest release of Gateway API is shown here. Past Gateway API implementations and feature support can be found here:
+
+    - [v1.0.0](./implementation-table-v1.0.0.md)
+"""
+
+
 def generate_conformance_tables(reports):
 
-    gateway_http_table = generate_profiles_report(reports, 'HTTP')
+    gateway_http_table = generate_profiles_report(reports, 'GATEWAY-HTTP')
     gateway_http_table = gateway_http_table.rename_axis('Organization')
 
-    # Currently no implementation has extended supported features listed.
-    # Can uncomment once a list is needed to keep track
-    # gateway_tls_table = generate_profiles_report(reprots,'TLS')
+    gateway_tls_table = generate_profiles_report(reports, 'GATEWAY-TLS')
+    gateway_tls_table = gateway_tls_table.rename_axis('Organization')
 
-    mesh_http_table = generate_profiles_report(reports, 'MESH')
+    gateway_grpc_table = generate_profiles_report(reports, 'GATEWAY-GRPC')
+    gateway_grpc_table = gateway_grpc_table.rename_axis('Organization')
+
+    mesh_http_table = generate_profiles_report(reports, 'MESH-HTTP')
     mesh_http_table = mesh_http_table.rename_axis('Organization')
 
     with open('site-src/implementation-table.md', 'w') as f:
+
         f.write(desc)
         f.write("\n\n")
 
         f.write(warning_text)
         f.write("\n\n")
 
+        f.write(past_versions)
+        f.write("\n\n")
+
         f.write("## Gateway Profile\n\n")
         f.write("### HTTPRoute\n\n")
         f.write(gateway_http_table.to_markdown()+'\n\n')
+        f.write('### TLSRoute\n\n')
+        f.write(gateway_tls_table.to_markdown()+'\n\n')
+        f.write('### GRPCRoute\n\n')
+        f.write(gateway_grpc_table.to_markdown()+'\n\n')
```

Comment:
Tiny nit: I'd recommend ordering these in terms of stability, and then by the length they've been GA - so HTTPRoute > GRPCRoute > TLSRoute.